### ISSUE TITLE
[Feature Request] Possibility to set rounded corners in the theme editor file

### ISSUE THREAD

**Issue by LTMX** [unitycoder/UnityLauncherPro#115]

**Comment by unitycoder** [comment]
yeah, could be added to custom theme settings, if its possible with WPF.

Is that rounded corners for window or for all elements (buttons, tabs, datagrid and so on?)

**Comment by LTMX** [comment]
> yeah, could be added to custom theme settings, if its possible with WPF.
> 
> Is that rounded corners for window or for all elements (buttons, tabs, datagrid and so on?)

Yes I envisioning every UI element, rounded... And if the window itself can, I'de get that too !

**Comment by unitycoder** [comment]
testing rounded buttons, if i can make it optional, could be possible to add. (and if there is some interest)

![image](https://github.com/user-attachments/assets/772ebfd7-466e-4719-8c5d-46cdce789caa)

![image](https://github.com/user-attachments/assets/e9a8684d-4671-4d58-9ddf-531befb9eb7c)

**Comment by Pituivan** [comment]
That looks much more awesome
